R/two-thirds-insulin.R

```diff
@@ -0,0 +1,57 @@
+#' Add column indicating if at least two-thirds of GLD doses are insulin doses.
```

Comment:
```suggestion
#' Add column indicating if at least two-thirds of GLD doses are insulin doses
```
Bc they don't use periods at the end in the examples in the [R Packages](https://r-pkgs.org/man.html#sec-man-workflow) book.